static/app/components/events/interfaces/performance/spanEvidence.tsx

```diff
@@ -20,26 +20,30 @@ export type TraceContextSpanProxy = Omit<TraceContextType, 'span_id'> & {
   span_id: string; // TODO: Remove this temporary type.
 };
 
+function getEvidenceDescription(event: EventTransaction) {
+  if (event.perfProblem?.type === 1008) {
```

Comment:
I see you added an enum for this group type called `PERFORMANCE_FILE_IO_MAIN_THREAD`, can we use that here so it's clearer that this number is correct?